app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,26 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
```

Comment:
You don't need `ConstraintLayout` here. Because you have only bottom-nav and fragment-container. So, `LinearLayout` is totally enough